plugins/modules/aci_rest.py

```diff
@@ -405,7 +412,13 @@ def main():
         rsp_subtree_preserve=dict(type="bool", default=False),
         page_size=dict(type="int"),
         page=dict(type="int"),
-        convert_values_to_str=dict(type="bool", default=True),  # To support Jinja2 3.1.6 and later versions.
+        # To support Ansible Core 2.19.0 and later, Jinja2 3.1.6 and later versions.
+        # The primitives_to_string parameter defaults to True to maintain compatibility with Ansible Core versions earlier than 2.19.0.
+        primitives_to_string=dict(
```

Comment:
Is `primitives_to_string` meant to default to True as it mentions in the comment? I can't see a default in the spec.